src/test/response_cache_test.cc

```diff
@@ -316,21 +318,39 @@ check_status(tc::Status status)
 }
 
 void
-cache_stats(std::unique_ptr<tc::RequestResponseCache>& cache)
+insert(
```

Comment:
I just noticed this. I think it would be good to be consistent with the style everywhere (i.e., use camel case).

Reply:
done